src/flag_gems/ops/log_softmax.py

```diff
@@ -58,6 +60,96 @@ def log_softmax_kernel(
     tl.store(output_ptrs, softmax_output, mask=mask)
 
 
+@libentry()
```

Comment:
We have a new implementation in https://github.com/FlagOpen/FlagGems/pull/76, which improves performance a lot, maybe you can test that too.